### DISCUSSION TITLE
Twilio console - email templates

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#918] [Q&A] [answered]
Hello there,

Do we have templates / components as part of paste and sketch for Twilio product specific emails?
For example - reset password / invite user / billing invoice is ready / order is fulfilled / batch action is completed — with error.

We are building a feature where customers gets notified when bulk action is completed.
![email-template](https://user-images.githubusercontent.com/69016180/98424535-cfbcf980-2046-11eb-9424-fb861fcef5ba.png)

Any guidance on how to improve the design or components to use is greatly appreciated.

Thank you.

**Comment by aayushpi** [comment, marked as answer]
Hi @ishah-twilio, we don't provide any guidance on product emails at this time. 

As a general principle, I would aim to follow our UX principles, particularly "[develop a seamless experience](https://paste.twilio.design/principles#develop-a-seamless-experience)", and try to match visual elements and content tone of voice so that the transition from email to a logged in experience is smooth and natural.
